### PR TITLE
Move tombstone metadata to each schema

### PR DESCRIPTION
**Pull request by southeo**
- update pidStatus values
- add tombstone metadata to each fdo profile instead of having it as a separate profile to align with practices in DiSSCo

### CODE REVIEW

**Review by sharifX — CHANGES_REQUESTED** [review]

**Review by sharifX — APPROVED** [review]

File: data-model/fdo-type/shared-model/0.3.0/examples/tombstone-metadata-example.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "@type": "ods:Tombstone",
   "ods:tombstonedDate": "2023-06-15T14:30:00Z",
```
Comment (samleeflang) [inline data-model/fdo-type/shared-model/0.3.0/examples/tombstone-metadata-example.json:3]: Shouldn't this also be `ods:tombstoneDate` and the `ods:tombstoneByAgent` ?

File: data-model/fdo-profile/annotation/0.1.0/examples/example-tombstoned-annotation-fdo.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "pidRecordIssueDate": "2023-11-17T11:57:02.933Z",
+  "pidRecordIssueNumber": 1,
+  "structuralType": "digital",
+  "pidStatus": "TOMBSTONE",
+  "targetPid": "https://hdl.handle.net/20.500.125/1111111",
+  "targetType": "DigitalSpecimen",
+  "motivation": "oa:commenting",
+  "annotationHash": "faab142f-40a5-fc15-7085-49c23301c608",
+  "ods:tombstoneText": "Annotation deleted by user",
+  "ods:hasRelatedPID": [
```
Comment (samleeflang) [inline data-model/fdo-profile/annotation/0.1.0/examples/example-tombstoned-annotation-fdo.json:21]: Does indeed look stange, some terms got namespace, some don't...

File: data-model/fdo-profile/digital-media/0.1.0/examples/example-tombstoned-digital-media-fdo.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  "digitalObjectName": "Digital Media Object",
+  "pid": "https://hdl.handle.net/TEST/J7C-4MC-1HK",
+  "pidIssuer": "https://ror.org/04wxnsj81",
+  "pidIssuerName": "DataCite",
+  "issuedForAgent": "https://ror.org/0566bfb96",
+  "issuedForAgentName": "Naturalis Biodiversity Center",
+  "pidRecordIssueDate": "2023-10-16T11:32:10.828Z",
+  "pidRecordIssueNumber": 1,
+  "structuralType": "digital",
+  "pidStatus": "TOMBSTONE",
```
Comment (samleeflang) [inline data-model/fdo-profile/digital-media/0.1.0/examples/example-tombstoned-digital-media-fdo.json:15]: Might missed it but there should be tombstone data and text somewhere right?

File: data-model/fdo-profile/digital-media/0.1.0/examples/example-digital-media.json [hunk 4]
```diff
@@ -30,5 +30,7 @@
   "licenseUrl": "cc-by-nc-nd",
   "rightsholderName": "National Museum of Natural History",
   "rightsholderPid": "https://ror.org/03wkt5x30",
-  "rightsholderPidType": "Resolvable"
+  "rightsholderPidType": "Resolvable",
+  "ods:tombstoneText": "Resource deleted by admin",
```
Comment (samleeflang) [inline data-model/fdo-profile/digital-media/0.1.0/examples/example-digital-media.json:34]: I think these should be in the next file

File: data-model/fdo-profile/digital-media/0.1.0/schema/digital-media.json [hunk 5]
```diff
@@ -102,6 +98,53 @@
...
+                "ods:isDuplicateOf"
+              ]
+            }
+          },
+          "additionalProperties": false
+        },
+        "$comment": "idx = 31"
+      }
+    },
+    "tombstonedDate": {
```
Comment (samleeflang) [inline data-model/fdo-profile/digital-media/0.1.0/schema/digital-media.json:139]: Maybe at least make this `ods` as well

File: data-model/fdo-profile/doi-kernel/0.1.0/examples/example-tombstoned-doi.json
Comment (samleeflang) [inline data-model/fdo-profile/doi-kernel/0.1.0/examples/example-tombstoned-doi.json:1]: Missing tombstone info here, status is also ACTIVE

**Review by samleeflang — COMMENTED** [review]
Couple comments main ones:
- Is not also `ods:TombstoneData` so all terms without the `d` (this gets me so confussed) :roll_eyes: 
- Namespaces is an issue, maybe at least namespace all tombstone terms in fdo (so also `tombstonedDate`

File: data-model/fdo-type/shared-model/0.3.0/examples/tombstone-metadata-example.json (hunk 1 above)
Comment (southeo) [reply data-model/fdo-type/shared-model/0.3.0/examples/tombstone-metadata-example.json:3]: I did a replace all :upside_down_face: reverted

File: data-model/fdo-profile/annotation/0.1.0/examples/example-tombstoned-annotation-fdo.json (hunk 2 above)
Comment (southeo) [reply data-model/fdo-profile/annotation/0.1.0/examples/example-tombstoned-annotation-fdo.json]: removed the `ods` on the fdo attributes

File: data-model/fdo-profile/digital-media/0.1.0/schema/digital-media.json (hunk 5 above)
Comment (southeo) [reply data-model/fdo-profile/digital-media/0.1.0/schema/digital-media.json]: no ods anywhere

**Review by samleeflang — APPROVED** [review]
:postal_horn:

**Pull request merged**